public/build/pdf.js

```diff
@@ -12318,34 +12345,54 @@
             container.addEventListener(
               "click",
               () => {
-                const can = document.getElementById("referenceDetailsView");
-                if (can.firstElementChild != null) {
-                  can.removeChild(can.firstElementChild);
+                const can = document.getElementsByClassName(
+                  "reference-view-wrapper",
+                )[0];
+                if (can != null) {
+                  can.parentNode.removeChild(can);
                 }
-                var canv = document.createElement("canvas");
-                canv.id = "referencedtl";
+
+                var canv = document.getElementsByClassName(
+                  "referenceview-canvas",
+                )[0];
+                canv.style.visibility = "visible";
                 page._transport.getDestination(data.dest).then((data) => {
                   let page_number = this.linkService._cachedPageNumber(data[0]);
+                  page._transport.getDestinations().then((res) => {
+                    var d = res["appendix.A"];
+                    if (d == undefined) {
+                      d = res["section.1"];
+                    }
+                    if (d == undefined) {
+                      d = 50;
+                    } else {
+                      d = d[2];
+                    }
+                    page._transport.getPage(page_number).then(function (page) {
+                      const rr = page.getViewport({ scale: 1 });
+                      const ll = data[3];
+                      const scalefact = 1;
```

Comment:
Please change the scalefact to 1.6, canvas resolution is much better when the sidebar is resized.